src/app/(with-main-header)/mypage/ClosetTab.tsx

```diff
@@ -1,33 +1,50 @@
 import Character from "@/components/common/Character";
 import { costumes } from "@/consts/costumes";
 import Image from "next/image";
-import BlueDividerLong from "@/assets/images/BlueDividerLong.svg";
+import WhiteDividerLong from "@/assets/images/WhiteDividerLong.svg";
 import basic from "@/assets/images/costumes/basic.png";
 import spotlight from "@/assets/images/spotlight.png";
+import WhiteDividerShort from "@/assets/images/WhiteDividerShort.svg";
 
 export default function ClosetTab() {
   return (
-    <div className="flex gap-17 px-8">
-      <div className="relative min-w-117 h-184">
+    <div className="flex flex-col lg:flex-row gap-0 lg:gap-17 px-8 overflow-y-auto">
+      <div className="relative w-full h-90 lg:w-117 lg:h-184 flex-shrink-0">
         <Image
           src={spotlight}
           alt="캐릭터를 비추는 스포트라이트 이미지"
-          className="-translate-y-20 h-184"
+          className="object-contain"
           sizes="100%"
           fill
         />
-        <Character costumeSrc={basic} className="h-119" />
+        <Character
+          costumeSrc={basic}
+          className="translate-y-[15%] h-58 lg:h-119"
+        />
       </div>
-      <section className="flex flex-col items-center w-full gap-8">
-        <Image src={BlueDividerLong} alt="파란 구분선 이미지" />
-        <section className="grid grid-cols-2 xl:grid-cols-3 2xl:grid-cols-4 auto-rows-auto w-full justify-items-center gap-6 h-120 overflow-auto">
+      <section className="flex flex-col items-center w-full gap-5 lg:gap-8 lg:mt-28">
+        <Image
+          src={WhiteDividerLong}
+          alt="흰 구분선 이미지"
+          className="hidden lg:block"
+        />
+        <Image
+          src={WhiteDividerShort}
+          alt="흰 구분선 이미지"
+          className="lg:hidden"
+        />
+        <section className="grid grid-cols-2 xl:grid-cols-3 2xl:grid-cols-4 auto-rows-auto w-full justify-items-center gap-4 lg:gap-6 h-120">
           {costumes.map((costume, index) => (
             <button
               key={`costume-${index}`}
-              className="flex items-end w-56 h-56"
+              className="w-full aspect-square max-w-[224px]"
```

Comment:
```suggestion
              className="w-full aspect-square max-w-56"
```
이제는 w나 h 같은 경우에는 모든 숫자를 넣을 수 있습니다! `max-w-56.5` 처럼도 사용할 수 있으니, 정말 필요한 경우가 아니면 임의값을 사용하지 않는 게 더 좋을 것 같아요!